src/video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -50,8 +50,7 @@ def analyze_video(video_path, time_range=None):
 
         # Add found frames to our dictionary
         if len(faces) > 0:
-            frame_number = int(cap.get(cv2.CAP_PROP_POS_FRAMES))
-            dict_of_faces[str(frame_number)] = faces
+            dict_of_faces[str(int(cap.get(cv2.CAP_PROP_POS_MSEC)))] = faces
```

Comment:
Maybe a comment to understand what is going on here. Might not be needed

Reply:
done